docs/userManual.md

```diff
@@ -888,7 +881,7 @@ axes of a plot, then the screen is divided into small patches a few
 pixels wide and the number of data points that falls within each patch
 is counted.  The number of values that falls within each patch is
 displayed as a heatmap, where the color intensity indicates the number
-of points.  A heatmap where the Y axis is not categorical will also
+of points. The mapping between the color intencity and the count could be in log scale if the range is large or linear if the range is smaller. Next to the mouse an overlay box displays the x value, the y value and the count. A heatmap where the Y axis is not categorical will also
```

Comment:
the overlay box is already described earlier in this section.

Reply:
Can you check whether this text is duplicated in this section?

Reply:
Next paragraph it says that a menu can toggle between log scale in linear scale. Here we say that the default depends on the data range. I don't find any other duplication.

Reply:
I am talking about the overlay box next to the mouse.